stores/facilitySearchbarStore.ts

```diff
@@ -0,0 +1,11 @@
+import { defineStore } from 'pinia'
+import { ref } from 'vue'
+import type { Facility } from '~/typedefs/gqlTypes'
+
+const initializeStore = () => {
```

Comment:
does this actually need to be a store instead of just local state within the searchbar component? I think local state should be fine here 🤔